tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -75,6 +75,7 @@ def hash_to_patterns(hash)
     if /^\[.*\]$/.match(value)
       value.gsub!(/[\[\]]/) { |s| '\\' + "#{s}" }.gsub!(/\"/) { |_s| '\'' }
     end
+    value.gsub!(/[\(\)]/) { |s| '\\' + "#{s}" } if /\(.*\)/.match(value)
```

Comment:
The version string is usually something of the sort '7.0(3)I2(5)'. The parenthesis in the string will cause Match of the generated regexp to fail. Hence need to escape those characters.